packages/core/src/components/image-overlay/image-overlay.template.ts

```diff
@@ -0,0 +1,37 @@
+import { html } from "lit-html";
+
+export function imageOverlayTemplate() {
```

Comment:
De `*Template()` functions zijn wrappers om voor één component. Maar deze `imageOverlayTemplate()` geeft ons nu geen `<dso-image-overlay> />`, maar 'n compleet Viewer Grid.

En eigenlijk heb je hier een soort van voorbeeldpagina gemaakt. Aangezien we daar nog hard over na aan het denken zijn, stel ik voor dat je `imageOverlayStories()` beperkt tot het enkele component. Je kan nog een image in `viewerGridStories()` plaatsen.

Reply:
Ja, dit blijft een terugkerend dilemma. Voor weergave van het component in Storybook voor de buitenwereld wil je eigenljik alleen maar, en enkel één component tonen (de showroom), maar voor ontwikkelen is een complexere weergave noodzakelijk.